Formula/lxc.rb

```diff
@@ -0,0 +1,23 @@
+class Lxc < Formula
+  desc "LXC: cli client for interacting with LXD"
```

Comment:
Please don’t include the formula name in the description: `desc "CLI client for interacting with LXD"`